MI/MIValue.cpp

```diff
@@ -122,6 +122,11 @@ void MIValue::SetArrayItem(const MIValue& value, MI_Uint32 index)
             strDest = new MI_Char[len];
             memcpy_s(strDest, len * sizeof(MI_Char), value.m_value.string, len * sizeof(MI_Char));
         }
+        else
+        {
+            strDest = new MI_Char[1];
```

Comment:
This generates a memory leak, since you don't deallocate the string. Just do: 
strDest = L"";